repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/filters/identityv3/cache/IdentityV3CacheOffSetOldTest.groovy

```diff
@@ -1,153 +0,0 @@
-                            ])
-                    mc.receivedResponse.code.equals("200")
-                    lastTokenValidation = DateTime.now()
-                }
-            }
-            clientThreads.add(thread)
-        }
-        clientThreads*.join()
-
-        then: "REPOSE should validate the token and then pass the request to the origin service"
-        fakeIdentityV3Service.validateTokenCount == uniqueUsers
-
-        when: "Same users send subsequent GET requests up to but not exceeding the token timeout - cache offset (since some requests may expire at that time)"
-        fakeIdentityV3Service.resetCounts()
-        DateTime minimumTokenExpiration = initialTokenValidation.plusMillis(tokenTimeout - cacheOffset)
-        clientThreads = new ArrayList<Thread>()
-
-        userTokens.eachWithIndex { token, index ->
-            def thread = Thread.start {
-                while (minimumTokenExpiration.isAfterNow()) {
-                    MessageChain mc = deproxy.makeRequest(
-                            url: reposeEndpoint, method: 'GET',
-                            headers: [
-                                    'content-type'   : 'application/json',
-                                    'X-Subject-Token': token
-                            ])
-                    mc.receivedResponse.code.equals("200")
-                }
-            }
-            clientThreads.add(thread)
-        }
-        clientThreads*.join()
-
-        then: "All calls should hit cache"
-        fakeIdentityV3Service.validateTokenCount == 0
-
-        when: "Cache has expired for all tokens (token timeout + cache offset), and new GETs are issued"
-        fakeIdentityV3Service.resetCounts()
-        DateTime maximumTokenExpiration = lastTokenValidation.plusMillis(tokenTimeout + cacheOffset)
-        //wait until max token expiration is reached
-        while (maximumTokenExpiration.isAfterNow()) {
-            sleep 200
-        }
-
-        clientThreads = new ArrayList<Thread>()
-
-        userTokens.eachWithIndex { token, index ->
-            def thread = Thread.start {
-                MessageChain mc = deproxy.makeRequest(
-                        url: reposeEndpoint, method: 'GET',
-                        headers: [
-                                'content-type'   : 'application/json',
-                                'X-Subject-Token': token
-                        ])
-                mc.receivedResponse.code.equals("200")
-            }
-            clientThreads.add(thread)
-        }
-        clientThreads*.join()
-
-        then: "All calls should hit identity"
-        //since we are talking about time based testing, we cannot always validate against a concrete number.  This is testing a range of requests.
-        fakeIdentityV3Service.validateTokenCount == uniqueUsers
-
-        where:
-        uniqueUsers | initialCallsPerUser
-        50          | 1
-    }
-}
```

Comment:
Should we keep this test and the other deleted test? It looks like they were testing the deprecated configuration style. While we got rid of the previously deprecated style, we introduced a newly deprecated style, so it seems like these would serve the same function they have been serving.

Reply:
We don't seem to consistently test this way and since the schemas are checked for this capability I went ahead and just removed the old configs.